circleci/github-release

```diff
@@ -5,12 +5,21 @@
 #
 # Usage:
 #
-#   github-release [GITHUB_TOKEN] [ARTIFACTS_DIR]
+#   github-release [--pre-release] <GITHUB_TOKEN> [ARTIFACTS_DIR]
```

Comment:
nit - keep `[]` around `GITHUB_TOKEN`. 

Also, is there any reason to not just have `--pre-release` as the 3rd option so we get rid of the special casing?

```bash
#!/bin/bash


echo $1 $2 $3
```
gives us
```
~/go/src/github.com/Clever/ci-scripts ❯❯❯ ./s.sh hi sayan
hi sayan

~/go/src/github.com/Clever/ci-scripts ❯❯❯ ./s.sh hi sayan --pre-release
hi sayan --pre-release
```

Reply:
Oops - keep the equality check as well ;)

Reply:
The `<>` means it's required, the documentation was incorrect before - GitHub token is always required. 

Moving it as the 3rd argument doesn't solve any of the special casing, since we still need to check for the case where they've left off the (optional) `ARGUMENTS_DIR`